kolibri/plugins/management/assets/src/device_management/views/index.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+
+  <core-base :topLevelPageName="DEVICE" :appBarTitle="$tr('deviceManagementTitle')">
```

Comment:
In other places we draw this from core constants `TopLevelPageNames`

ref new, independent issue: #2102